config/v1/types_cluster_version.go

```diff
@@ -50,6 +50,10 @@ type ClusterVersionSpec struct {
 	// +optional
 	DesiredUpdate *Update `json:"desiredUpdate,omitempty"`
 
+	// automaticUpdates enables automatic updates.
+	// +optional
+	AutomaticUpdates bool `json:"automaticUpdates,omitempty"`
```

Comment:
We want people to opt-out, this API is opt-in.

Reply:
> We want people to opt-out, this API is opt-in.

[Existing users will need to opt in][1].  How do we do that with an opt-out API?  Can we have an opt-in API here and have an opt-out API in the install-config, so new installers will generate ClusterVersion with this set `true` by default?

[1]: https://github.com/openshift/api/pull/326#discussion_r288770193

Reply:
The zero value of a bool is false so "auto-updates=false" means "no auto updates by default", thus it's opt in.

Reply:
It's opt in *here*, and I think we want that for compat with existing clusters (which are all manual-update).  If/when we add an installer knob that feeds this property, that installer knob should be opt out.  Are we all on the same page, or am I missing something?